SConstruct

```diff
@@ -502,7 +502,7 @@ libraries = {
 		"pythonEnvAppends" : {
 			"LIBS" : [ "GafferImageTest", "GafferImageBindings" ],
 		},
-		"additionalFiles" : glob.glob( "python/GafferImageTest/scripts/*" ),
+		"additionalFiles" : glob.glob( "python/GafferImageTest/scripts/*" ) + glob.glob( "python/GafferImageTest/images/*" ),
```

Comment:
Some of the images in there are needed for GafferTest. The ones that are purely for GafferImageTest should be moved at some point, yes, but I don't think it makes sense to do so in this PR...